graphql-protocol/src/main/java/com/linecorp/armeria/server/graphql/protocol/AbstractGraphqlService.java

```diff
@@ -150,6 +191,18 @@ protected HttpResponse doPost(ServiceRequestContext ctx, HttpRequest request) th
         return unsupportedMediaType();
     }
 
+    private void bindMultipartVariable(Map<String, Object> variables, List<String> value,
+                                       List<MultipartFile> multipartFiles) {
+        if (multipartFiles.isEmpty()) {
+            return;
+        }
+        for (MultipartFile multipartFile : multipartFiles) {
```

Comment:
Could you please explain a little more? 🤣

Reply:
From my understanding, `mapVariable(String objectPath, Map<String, Object> variables, MultipartFile multipartFile)` iterates through an `objectPath` through `variables`.

e.g.
```
objectPath=variables.a.b
variables={
  a: {
    b: null
  }
}
```
Becomes
```
variables={
  a: {
    b: multipartFile
  }
}
```

I thought that in this loop, each `multipartFile` will eventually be written to the path, and then overwritten until the last `multipartFile` is reached.
Let me know if I'm misunderstanding anything 😅

Reply:
I think multipartFile is the same as what you mentioned, because binding is actually handled under the following conditions.

```
if (i == segments.size() - 1)
```

Reply:
Sorry 😅   I should've made my intention more clear.

What do you think of modifying as follows?

```
private static void bindMultipartVariable(Map<String, Object> variables, List<String> operationsPaths,
                                          @Nullable List<MultipartFile> multipartFiles) {
    if (multipartFiles == null || multipartFiles.isEmpty()) {
        return;
    }
    final MultipartFile multipartFile = multipartFiles.get(0);
    for (String objectPath : operationsPaths) {
        MultipartVariableMapper.mapVariable(objectPath, variables, multipartFile);
    }
}
```

Reply:
ah...! understand. I was talking about something different each time. 🤣 

The specification is in the form of a list, but since there is no clear content, it seems okay to change it like this.